src/String.cpp

```diff
@@ -1534,7 +1534,21 @@ int String::compare(const ::String &inRHS) const
 

```

Comment:
(Cannot comment on the actual lines 1532 and 1533) There is now `String::isUTF16Encoded()`, which is an `inline` function that's basically identical to the flag check above, so `s0IsWIde` and `s1IsWide` can be defined in a more readable way.